src/affine.jl

```diff
@@ -1,7 +1,7 @@
 function linmap(mat, img::AbstractArray{T,N}, SD=Matrix(1.0*I,N,N), initial_tfm=IdentityTransformation()) where {T,N}
-    mat = [mat...]
-    SD = update_SD(SD, initial_tfm)
-    mat = SD\reshape(mat, N,N)*SD
+    mat = [mat...] #TODO WHY not put dimension mismatch here instead.
+    # SD = update_SD(SD, initial_tfm)
+    mat = SD\reshape(mat, N,N)*SD #TODO this is basically arrayscale. TODO improve commenting?
```

Comment:
If there's no translational component, yes. But notice:
```julia
julia> SD = [1 0; 0 2]
2×2 Array{Int64,2}:
 1  0
 0  2

julia> tfm = Translation(3, 3) ∘ LinearMap(RotMatrix(π/6))
AffineMap([0.8660254037844387 -0.49999999999999994; 0.49999999999999994 0.8660254037844387], [3, 3])

julia> arrayscale(tfm, SD)
AffineMap([0.8660254037844387 -0.9999999999999999; 0.24999999999999997 0.8660254037844387], [3.0, 1.5])
```
The translational component is different, too! The distinction is whether you're computing the translation in physical units or array-units. The new "standard" is to compute it in physical units (e.g., `μm` even if we're not passing in units).